pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -507,29 +507,34 @@ mod config {
 	}
 
 	#[test]
-	fn update_mint_fee_should_work() {
-		let original_fee = 550_000_000_000;
-		let update_fee = 650_000_000_000;
+	fn update_mint_fees_should_work() {
+		let original_fees =
+			MintFees { one: 550_000_000_000, three: 500_000_000_000, six: 450_000_000_000 };
+		let update_fees =
+			MintFees { one: 650_000_000_000, three: 600_000_000_000, six: 750_000_000_000 };
 
 		ExtBuilder::default().organizer(ALICE).build().execute_with(|| {
-			assert_eq!(AwesomeAvatars::mint_fee(), original_fee);
-			assert_ok!(AwesomeAvatars::update_mint_fee(Origin::signed(ALICE), update_fee));
-			assert_eq!(AwesomeAvatars::mint_fee(), update_fee);
+			assert_eq!(AwesomeAvatars::mint_fees(), original_fees);
+			assert_ok!(AwesomeAvatars::update_mint_fees(Origin::signed(ALICE), update_fees));
+			assert_eq!(AwesomeAvatars::mint_fees(), update_fees);
 			System::assert_last_event(mock::Event::AwesomeAvatars(crate::Event::UpdatedMintFee {
-				fee: update_fee,
+				fee: update_fees,
 			}));
 		});
 	}
 
 	#[test]
-	fn update_mint_fee_should_reject_non_organizer_as_caller() {
+	fn update_mint_fees_should_reject_non_organizer_as_caller() {
 		ExtBuilder::default().organizer(ALICE).build().execute_with(|| {
-			let original_mint_fee = AwesomeAvatars::mint_fee();
+			let original_mint_fees = AwesomeAvatars::mint_fees();
```

Comment:
The intention here (and following commet) is to check that the mint fee doesn't change after calling _update_mint_fees_ if the method failed. Just to be sure, should I delete this check, move to a different test case or delete?

Reply:
Oh I meant `assert_noop` already checks for all storage changes (it checks mint_fee and other storage changes).

[See here it checks for storage root changes](https://github.com/paritytech/substrate/blob/563f4820d8f36d256ada7ea3fef46b2e94c4cd5a/frame/support/src/lib.rs#L789-L803). If any of our data changes -> the root will change because it's a tree

Reply:
Great then, I'll remove these checks. Thanks!